File: website/devis.com/cgi-bin/scriptQuery.pl
#!/usr/bin/perl

use strict;
use warnings;
use CGI;


my $cgi = CGI->new;

my $query = $ENV{'QUERY_STRING'};
my $alive = $ENV{'KEEP_ALIVE'};
my $value1;
my $value2;

if ($query)
{
    my @pairs = split(/&/, $query);
    foreach my $pair (@pairs)
    {
        my ($key, $val) = split(/=/, $pair);
        if ($key eq "email")
        {
            $value1 = $val;
        }
        if ($key eq "adresse")
        {
            $value2 = $val;
        }
    }
}

my $body = "
<header>
<style>       
body {
    display: flex;
    justify-content: center;
    flex-direction: column;
    align-items: center;
    height: 100vh;
    font-family: 'Poppins', sans-serif;
    font-weight: 300;
    font-size: 15px;
    line-height: 1.7;
    color: #c4c3ca;
    background-color: #1f2029;
    overflow-x: hidden;
    margin: 0;
    }
</style>
</header>
<html>


<body>


<p>Your email address: $value1\n</p>
<p>Your adress: $value2</p>

</body>
</html>";



print $body;

File: website/devis.com/cgi-bin/scriptQuery.py
import os
from urllib.parse import parse_qs

query_string = os.environ.get('QUERY_STRING')
params = parse_qs(query_string)

param1 = params.get('email', [None])[0]
param2 = params.get('adresse', [None])[0]

body = """<HTML>
<HEAD>
<TITLE>Info Client</TITLE>
</HEAD>

<BODY>

<P>Thanks for uploading your photo!</P>
<P>Your email address: {param1}</P>
<P>Your adress : {param2}</P>
</BODY>
</HTML>"""



print(body)